actions/matrixsynapse

```diff
@@ -26,6 +26,8 @@ from ruamel.yaml import round_trip_dump, round_trip_load
 
 from plinth import action_utils
 
+CONFIG_FILE_PATH = '/etc/matrix-synapse/homeserver.yaml'
```

Comment:
Could you not even import this from plinth/modules/matrixsynapse/__init__.py? It's still an unnecessary duplication in my opinion...